docs/emr_deployment.md

```diff
@@ -0,0 +1,126 @@
+---
+id: emr_deployment
+title: GeoTrellis EMR deployment with GDAL
+sidebar_label: GeoTrellis EMR deployment with GDAL
+---
+
+GeoTrellis can be used with _EMR 6_. However, if there is a need in GDAL bindings usage there can be some certain complexities.
+This section describes the entire _EMR 6_ deployment and GDAL installation in a way that would satisfy GeoTrellis GDAL requirements.
+
+Starting _GeoTrellis_ `3.4.2`, _GeoTrellis_ is compatible only with the GDAL`3.1 +` versions.
+
+## Installing GDAL on EMR
+
+The EMR installation conatans of the following steps:
+
+1. Install _GDAL_
+2. Set Spark `LD_LIBRARY_PATH`. In this case `GeoTrellis GDAL` would be able to use the installed `GDAL`.
+
+You can also check out [GeoTrellis Spark template](https://github.com/geotrellis/geotrellis-spark-job.g8) that is properly configured for the EMR jobs.
+
+### EMR Bootsrap script (GDAL installation on EMR)
+
+The EMR bootstrap script with all appropriate settings is located under the following S3 URI: [s3://geotrellis-test/emr-gdal/bootstrap.sh](s3://geotrellis-test/emr-gdal/bootstrap.sh).
+It installs GDAL 3.1.2 (by default) through conda and sets all appropriate env variables in case 
+it would be required to log in onto nodes and experiment with the `spark-shell`.
+
+```bash
```

Comment:
I always go back and forth on this, but isn't pasting the script here redundant? Can't the user just download and inspect the bootstrap script on S3 we linked to above? This seems like a code block that will rapidly be broken and forgotten about.

Reply:
@CloudNiner I don't know what is the better option: mb some one wants to see what is done in some general sense? I can completely remove it.

Reply:
Yeah I think if someone wants to see the exact steps they can pull the script and inspect it themselves. I just don't see this, as a copied complete script inline being worth the maintenance burden. I'd vote pretty strongly for just removing it.

Reply:
@CloudNiner sounds good to me, since I don't have strong feelings about it.